### PR TITLE
Added an onClick event on user's profile that navigates to the user's…

### PR DESCRIPTION
**Pull request by JM066**
… projects

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ x] New and existing unit tests pass locally with my changes
- [ x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/web/src/pages/Project/index.tsx [hunk 1]
```diff
@@ -47,10 +47,16 @@ function Project() {
         <Header>
           <InfoBox>
             <StyledAvatar
+              onClick={() => {
```
Comment (ivanms1) [inline apps/web/src/pages/Project/index.tsx:50]: could you wrap the icon inside an unstyled button, for accessibility purposes.

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/Project/index.tsx (hunk 1 above)
Comment (JM066) [reply apps/web/src/pages/Project/index.tsx:50]: Ok! I will also add the functionality to navigate back to previous page w/o refreshing

File: apps/web/src/pages/Project/index.tsx [hunk 2]
```diff
@@ -40,17 +41,41 @@ function Project() {
 
   return (
     <>
-      <CloseButton onClick={() => router.back()} variant='ghost'>
+      <CloseButton
+        onClick={() =>
+          router.push({
+            pathname: previous as string,
+          })
```
Comment (ivanms1) [inline apps/web/src/pages/Project/index.tsx:48]:
```suggestion
         if(previous) {
           router.push({
            pathname: previous,
          })
          } else {
             router.push('/')
          }
```

**Review by ivanms1 — APPROVED** [review]
LGTM, just some suggestions

File: apps/web/src/pages/Project/index.tsx [hunk 3]
```diff
@@ -40,17 +41,41 @@ function Project() {
...
+        variant='ghost'
+      >
         <StyledCloseIcon />
       </CloseButton>
-      <Modal isOpen onClose={() => router.back()} className={modalStyles()}>
+
+      <Modal
+        isOpen
+        onClose={() =>
+          router.push({
```
[the remark this answers is not in the document]
Comment (JM066) [reply apps/web/src/pages/Project/index.tsx:58]: Right! Just changed it. Thanks

**Pull request merged**